chart/slackconnector/README.md

```diff
@@ -0,0 +1,6 @@
+# Slack Connector <!-- omit in toc -->
+
+## Overview
+
+The chart creates a Slack Connector Deployment and a Namespace in Kyma.
+Moreover it creates an application and a service which is binded to the newly created namespace.
```

Comment:
```suggestion
Moreover, it creates an application and a service which is bound to the newly created Namespace.
```

Regarding `is`: is it just the service that IS bound, or is it both the service AND the application that ARE bound?

Reply:
Only the service, you have to bind the application yourself